brew_elasticsearch.txt

```diff
@@ -0,0 +1,19 @@
+elasticsearch: stable 2.3.1, HEAD
```

Comment:
uh, oops. Indeed, never meant to be committed. 
Basically some output of brew info <package> so I know how to start the services.